bluepyemodel/icselector/modules/model_selector.py

```diff
@@ -0,0 +1,156 @@
+"""Model selector selects NEURON mechanisms for cell model configuration"""
+
+import copy
+
+from .mechanism import Mechanism
+
+
+class ModelSelector:
+    """selects NEURON mechanisms for cell model configuration"""
```

Comment:
small typo: missing capital letter on first word: selects -> Selects

Reply:
Done in [942a287](https://github.com/BlueBrain/BluePyEModel/pull/144/commits/942a287eb332cb58b5608511616eefcd2e76e880)